src/pager.js

```diff
@@ -1,122 +1,75 @@
-import {bindable, customElement, bindingMode} from 'aurelia-framework';
+import {bindable, customElement, bindingMode, computedFrom} from 'aurelia-framework';
 import {resolvedView} from 'aurelia-view-manager';
 
+@resolvedView('spoonx/pager', 'pager')
 @customElement('pager')
-@resolvedView('aurelia-pager', 'pager')
 export class Pager {
-  @bindable({defaultBindingMode: bindingMode.twoWay})
-
-  @bindable page      = 1;  // current page
-  @bindable pagerange = 3;  // ranges of pages to view e.g "3 4 [5] 6 7"
-  @bindable limit     = 30; // the amount of records on a page
-  @bindable criteria  = {}; // search criteria for DB resource
-  @bindable resource;       // data resource, either a ORM or a array
-  @bindable pages;          // total amount of pages
-
-  attached() {
-    if (!this.page) {
-      this.page = 1;
-    }
 
-    this.reloadCount();
+  @bindable({defaultBindingMode: bindingMode.twoWay}) page
+
+  @bindable data    //set the data if you want to alter the
+  @bindable range
+  @bindable limit
+  @bindable pages
+
+  constructor() {
+    this.page = this.page || 1;
   }
 
-  reloadCount() {
```

Comment:
How do i tell pager to reload the data based on the new criteria? You removed `reloadCount`.